docs/_static/auth/iam.yaml

```diff
@@ -355,30 +403,103 @@ paths:
+        * 503 - the authorizer could not be reached or returned an error
+      tags:
+        - auth
+      parameters:
+        - name: did
+          in: path
+          required: true
+          description: The DID of the requester, a Wallet owner at this node.
+          schema:
+            type: string
+            example: did:web:example.com
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              required:
+                - redirectURL
+                - scope
+                - userID
+                - verifier
+              properties:
+                verifier:
+                  type: string
+                  example: did:web:example.com
+                scope:
+                  type: string
+                  description: The scope that will be the service for which this access token can be used.
+                  example: eOverdracht-sender
                 userID:
                   type: string
-                  description: | 
+                  description: |
                     The ID of the user for which this access token is requested.
                 redirectURL:
                   type: string
                   description: |
                     The URL to which the user-agent will be redirected after the authorization request.
       responses:
         '200':
-          description: Successful request. Responds with an access token as described in rfc6749 section 5.1.
+          description: | 
+            Successful request. Responds with a redirectURL for the user and a token for the calling application.
+            The token can be used by the calling application to get the status of the flow.
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/TokenResponse'
-        '302':
-          description: |
-            The user-agent is redirected to the authorization server to start the OpenID4VP flow.
-            Optionally, the user could be asked to agree or authenticate.
-          headers:
-            Location:
+                $ref: '#/components/schemas/RedirectResponseWithToken'
+        default:
+          $ref: '../common/error_response.yaml'
+  /internal/auth/v2/accesstoken/{token}:
+    get:
+      operationId: retrieveAccessToken
+      summary: Get the access-token from the Nuts node that was request by /request-user-access-token.
+      description: |
+        If the OAuth2 flow has completed, this call will return the access token. 
+        After returning the access token, the token used to retrieve the access token is no longer valid.
+        If the flow hasn't completed yet, the response will contain the 'pending' status value.
+        Therefore this API can be used to poll for the status of the OAuth2 flow.
+        When the
+      tags:
+        - auth
+      parameters:
+        - name: token
```

Comment:
calling this `token` is kinda confusing, since it makes me think of OAuth2 (but it isn't) and the name doesn't specify what it exactly is (like access token, refresh token, etc). In the domain model it's called "FlowToken" which is clearer, but not that nice either. Maybe choose something else than token? Like ID/SessionID/FlowID/Handle/etc

Reply:
I thought `token` would make it clear that the dumber used is to be kept secure. But since it's only used in the backchannel communication between calling app and node (which is already secure). We can call it SessionID.